atc/metric/emitter/prometheus.go

```diff
@@ -797,8 +884,30 @@ func (emitter *PrometheusEmitter) checkMetric(logger lager.Logger, event metric.
 		logger.Error("failed-to-find-check-name-in-event", fmt.Errorf("expected check_name to exist in event.Attributes"))
 		return
 	}
+	checkStatus, exists := event.Attributes["check_status"]
+	if !exists {
+		logger.Error("failed-to-find-check-status-in-event", fmt.Errorf("expected check_status to exist in event.Attributes"))
+		return
+	}
+
+	emitter.checksVec.WithLabelValues(scopeID, checkName, checkStatus).Inc()
+
+	switch checkStatus {
```

Comment:
rather than going with 3 metrics, we could actually go with a single one - a histogram will always produce a `total`, and  given that we're already passing `checkStatus`, one would be able to do like `concourse_lidar_checks_duration_count`:

```go
// github.com/prometheus/client_golang/prometheus/histogram.go
//
func (h *histogram) Observe(v float64) {
        // ...

	// Increment count last as we take it as a signal that the observation
	// is complete.
	atomic.AddUint64(&hotCounts.count, 1)
}
```
(https://github.com/prometheus/client_golang/blob/333f01cef0d61f9ef05ada3d94e00e69c8d5cdda/prometheus/histogram.go#L291-L293)

given that histograms will also generate `N` timeseries  (being `N` the size of the number of buckets) which gets multiplied by the number of other possible labels, it'd be good to not have `scopeID` and `checkName` for this one then

Reply:
I think from comment https://github.com/concourse/concourse/pull/4692#pullrequestreview-311275172 `scopeID` is useful still? 

`checkName` is not a must indeed.